.eslintrc.js

```diff
@@ -41,5 +41,11 @@ module.exports = {
             ignoreTemplateLiterals: true,
             ignoreRegExpLiterals: true,
         }],
+        'import/prefer-default-export': [
+            ('off'),
+            { target: 'any' },
```

Comment:
this parameter is option in our case
you can just switch the rule off and that's it